managed/services/agents/vmagent.go

```diff
@@ -16,19 +16,28 @@
 package agents
 
 import (
+	"os"
 	"sort"
 
 	"github.com/percona/pmm/api/agentpb"
 	"github.com/percona/pmm/api/inventorypb"
 )
 
+var maxScrapeSizeEnv = "PMM_PROMSCRAPE_MAX_SCRAPE_SIZE"
+
 // vmAgentConfig returns desired configuration of vmagent process.
 func vmAgentConfig(scrapeCfg string) *agentpb.SetStateRequest_AgentProcess {
+	maxScrapeSize := "64MiB"
```

Comment:
Default value in function is not intuitive for me. Maybe we should consider moving it to the top of file as global variable and naming it with `defaultMaxScrapSize`?